docs/source/contrib/sqlalchemy.rst

```diff
@@ -0,0 +1,155 @@
+.. _sqlalchemy:
+
+Flask-SQLAlchemy
+=================
+
+Arrested comes with built in support for working with Flask-SQLAlchemy.  The SQLAlchemy mixin's provide automatic handling of commiting sessions when saving objects,
```

Comment:
Arrested comes with built in support for working with Flask-SQLAlchemy. The SQLAlchemy mixins support automatically committing the database session when saving objects, filtering Queries by a model's primary key when fetching single objects and the removal of objects from the database when deleting.  Simply put, the SQLAlchemy mixin takes care of the boring stuff when integrating Arrested with SQLAlchemy.